pkg/document/json/primitive_test.go

```diff
@@ -17,12 +17,14 @@
 package json_test
 
 import (
-	"testing"
-
 	"github.com/stretchr/testify/assert"
 
 	"github.com/yorkie-team/yorkie/pkg/document/json"
 	"github.com/yorkie-team/yorkie/pkg/document/time"
+
+	"math"
+	"testing"
+	gotime "time"
```

Comment:
```suggestion
	"math"
	"testing"
	gotime "time"
	
	"github.com/stretchr/testify/assert"

	"github.com/yorkie-team/yorkie/pkg/document/json"
	"github.com/yorkie-team/yorkie/pkg/document/time"
```

Please group in the order of **Go packages** and **outer packages** and **inner packages**.